service/ziti-tunnel/service/state.go

```diff
@@ -301,6 +301,7 @@ func (t *RuntimeState) LoadIdentity(id *Id, refreshInterval int) {
 	}
 
 	id.CId = cziti.NewZid(sc)
+	id.CId.Fingerprint = id.FingerPrint
```

Comment:
not sure this belongs here. why do you need to set this? - this should be coming from the csdk in the callback

Reply:
I dont think c-sdk maintains the fingerprint. This field is set in the zid.StatusChanges(int(status)) function, and this is called only after the controller event is received. I will remove the assignment, move the broadcast events operation after the StatusChanges function call.

Reply:
when the identity is loaded either from cache or from the c sdk the fingerprint will be set. This is the wrong place to do it. see https://github.com/openziti/desktop-edge-win/blob/ca7b21447164ad99d7f9ed9a93948e490aa59fd1/service/ziti-tunnel/service/state.go#L273

Reply:
I just removed the fingerprint assignment that I added